src/simplify.jl

```diff
@@ -8,19 +8,29 @@ these rules are `SymbolicUtils.SIMPLIFY_RULES`. If `fixpoint=true`
 repeatedly applies the set of rules until there are no changes.
 Applies them once if `fixpoint=false`.
 """
-function simplify(x, rules=SIMPLIFY_RULES; fixpoint=true)
+function simplify(x, rules::Vector; fixpoint=true)
     if fixpoint
         SymbolicUtils.fixpoint(RuleSet(rules))(x)
     else
         RuleSet(rules)(x)
     end
 end
```

Comment:
did you leave this for backwards compatibility? Should probably call the below method directly.

Reply:
yeah, makes sense.